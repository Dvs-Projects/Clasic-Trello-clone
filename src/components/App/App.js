import './App.css';

const App = () => {
  return (
   <p>Project driver</p>
  );
};

export default App;
