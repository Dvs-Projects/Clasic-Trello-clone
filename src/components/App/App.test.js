import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import App from './App';

describe('<App /> Test', () => {
  const wrapper = shallow(<App />);
  test('Should take snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
