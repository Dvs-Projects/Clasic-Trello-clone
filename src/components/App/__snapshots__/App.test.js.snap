// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Test Should take snapshot 1`] = `
<p>
  Project driver
</p>
`;
